Source/URedis/Public/URedis.h

```diff
@@ -19,6 +19,10 @@ class UREDIS_API FURedis final : public IModuleInterface {
 
     FString Ping(TOptional<FString> message = NullOpt) const;
 
+    bool Set(FStringView key, FStringView value) const;
+
+    TOptional<FString> Get(FStringView key) const;
```

Comment:
please add documentation to all the public methods, preferably using oxygen syntax